kool-core/src/jsMain/kotlin/de/fabmax/kool/platform/webgl/LoadedTextureWebGl.kt

```diff
@@ -64,6 +71,21 @@ class LoadedTextureWebGl(val ctx: JsContext, val target: Int, val texture: WebGL
         }
     }
 
+    override fun readTexturePixels(targetData: TextureData) {
+        with(ctx.gl) {
+            val fb = createFramebuffer()
```

Comment:
I think the framebuffer should be deleted after usage by calling `gl.deteFramebuffer(fb)`